src/vm/functions/crypto.rs

```diff
@@ -80,11 +80,11 @@ pub fn special_principal_of(
     let pub_key = match param0 {
         Value::Sequence(SequenceData::Buffer(BuffData { ref data })) => {
             if data.len() != 33 {
-                return Err(CheckErrors::TypeValueError(BUFF_33, param0).into());
+                return Err(CheckErrors::TypeValueError(BUFF_33.clone(), param0).into());
```

Comment:
Why do these become clone? Was it copy before? Is copy better? Can we make this copyable?

Reply:
Previously, BUFF_33 was a const, which means the line here was an inlined constructor, so more or less equivalent to an auto-impl'd clone. But since it's no longer a const, the clone needs to be explicit.

TypeSignature cannot implement `Copy` -- `Copy` works for types that can be safely `memcpy`ed without changing their semantics. That's not true of TypeSignatures (mostly because it's a recursive type).